src/main/java/algorithms/curated170/medium/parallelcourses/ParallelCoursesBFS.java

```diff
@@ -50,7 +46,7 @@ private int sortTopologically() {
     }
 
     private void createMap(int[][] courses) {
-
+        inDegreeMap = new int[numOfCourses + 1];
         courseChiMap = new LinkedList<>();
```

Comment:
We should convert this to:
     courseChiMap = new ArrayList<>();
 
 Otherwise, it will take 85ms instead of 8ms.